### ISSUE TITLE
deis ps:scale cmd=3 spawns all on same node

### ISSUE THREAD

**Issue by haf** [deis/deis#3590]
I have three machines, and I wanted to scale out so that I could eliminate a single machine misbehaving.

`deis scale cmd=3` spawns all three containers on the same node.

`deisctl list` tells everything is fine.

**Comment by aledbf** [comment]
@haf you cannot control in which node/s a process is scheduled (fleet "fault")
similar to https://github.com/deis/deis/issues/3289

**Comment by haf** [comment]
That seems like a pretty bad thing for a PaaS... Are you sure?

**Comment by carmstrong** [comment]
> That seems like a pretty bad thing for a PaaS... Are you sure?

Deis talks directly to the application unit scheduler (fleet in this case) and asks it to schedule the units. fleet's rather rudimentary scheduling decisions are one of the reasons we're spending the resources to investigate and integrate with other schedulers - see #3134 and #2744.

Stay tuned!

**Comment by haf** [comment]
Sorry, I'm gonna use this as a support-ticket, to the extent you'll allow me.

Since I didn't like the outcome, but figure it's at least random (?) scheduling, I tried scale cmd=1 and then scale cmd=3 again. That last cmd gave me 503 Service Unavailable and this `deis info` output:

```
=== app-com Processes

--- cmd:
cmd.1 up (v28)
cmd.2 created (v28)
cmd.3 created (v28)
```

**Comment by bacongobbler** [comment]
Are you on fleetd v0.9.1 or are you using v0.9.2 which we shipped with v1.5.2? that behaviour looks very familiar to the bugs we saw when doing release testing on v0.9.1.

**Comment by carmstrong** [comment]
@haf If you can collect the logs from the controller when you performed that scale, it should show the stacktrace.
